common/rpc/RequestBuilder.java

```diff
@@ -127,24 +127,39 @@ public static ClusterUserProto.ClusterUserManager.Create.Req createReq(String us
                 return ClusterUserProto.ClusterUserManager.Create.Req.newBuilder().setUsername(username).setPassword(password).build();
             }
 
+            public static ClusterUserProto.ClusterUserManager.Delete.Req deleteReq(String username) {
+                return ClusterUserProto.ClusterUserManager.Delete.Req.newBuilder().setUsername(username).build();
+            }
+
             public static ClusterUserProto.ClusterUserManager.All.Req allReq() {
                 return ClusterUserProto.ClusterUserManager.All.Req.newBuilder().build();
             }
+
+            public static ClusterUserProto.ClusterUserManager.PasswordSet.Req passwordSetReq(String username, String password) {
+                return ClusterUserProto.ClusterUserManager.PasswordSet.Req.newBuilder()
+                        .setUsername(username)
+                        .setPassword(password)
+                        .build();
+            }
+
+            public static ClusterUserProto.ClusterUserManager.Get.Req getReq(String username) {
+                return ClusterUserProto.ClusterUserManager.Get.Req.newBuilder().setUsername(username).build();
+            }
         }
 
         public static class User {
 
-            public static ClusterUserProto.ClusterUser.Password.Req passwordReq(String username, String password) {
-                return ClusterUserProto.ClusterUser.Password.Req.newBuilder().setUsername(username).setPassword(password).build();
+            public static ClusterUserProto.ClusterUser.PasswordUpdate.Req passwordUpdateReq(String username, String oldPassword, String newPassword) {
+                return ClusterUserProto.ClusterUser.PasswordUpdate.Req.newBuilder()
+                        .setUsername(username)
+                        .setOldPassword(oldPassword)
+                        .setNewPassword(newPassword)
+                        .build();
```

Comment:
if you notice our style is to inline things that fit in one line without overflow

Reply:
This does overflow all on one line unfortunately. Something like this?
```
return ClusterUserProto.ClusterUserManager.PasswordSet.Req.newBuilder()
                        .setUsername(username).setPassword(password).build();
```